app/src/main/java/app/revanced/manager/ui/screen/settings/DownloadsSettingsScreen.kt

```diff
@@ -229,10 +241,27 @@ fun DownloadsSettingsScreen(
     }
 }
 
+@Preview
+@Composable
+private fun PreviewTrustDialog() {
+    TrustDialog(
+        title = R.string.downloader_plugin_trust_dialog_title,
+        body = stringResource(
+            R.string.downloader_plugin_trust_dialog_body,
+        ),
+        onDismiss = { },
+        onConfirm = { },
+        pluginName = "app.revanced.manager.apkmirror",
+        signature = "23 01 84 F6 0B AE 2F EA F2 44 F1 0A 8B AC 05 3C 8F F3 3A 18 3B CC 36 5B 4D 8B 87 6D 2B 7F 48 09"
```

Comment:
While not stated in my RVMC/Weberie variant, you can use semi-colon ( : ) as separator like

2F:EA:F2:44 ... Like MAC Address

Reply:
That's not really typical display of signatures